lib/elixir/lib/uri.ex

```diff
@@ -29,6 +29,15 @@ defmodule URI do
 
   import Bitwise
 
+  format_charlist = fn [first | rest] ->
+    Enum.reduce(rest, <<?`, first, ?`>>, fn char, acc ->
+      acc <> <<?,, ?\s, ?`, char, ?`>>
+    end)
+  end
+
+  @reserved_characters ':/?#[]@!$&\'()*+,;='
+  @formatted_reserved_characters format_charlist.(@reserved_characters)
```

Comment:
```suggestion
  @formatted_reserved_characters Enum.map_join(@reserved_characters, ", ", &"`#{&1}`")
```

Reply:
this is printing the codepoint
> "`58`, `47`, `63`, `35`, `91`, `93`, `64`, `33`, `36`, `38`, `39`, `40`, `41`, `42`, `43`, `44`, `59`, `61`"

Reply:
it should be
``` 
Enum.map_join(reserved_characters, ", ", &"`#{<<&1>>}`")
```

Reply:
Just out of curiosity, is there another way to get the string representation of a codepoint?

Reply:
updated

Reply:
Or: 

    Enum.map_join(reserved_characters, ", ", &<<?`, &1, ?`>>)